app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/presenter/DummyUIPresenter.kt

```diff
@@ -20,4 +21,13 @@ class DummyUIPresenter: UIPresenter {
     override suspend fun getPicture(categoryString: String): CategorizedPicture? {
```

Comment:
Can we take the opportunity to solve this warning?
```
The corresponding parameter in the supertype 'UIPresenter' is named 'categoryName'. This may cause problems when calling this function with named arguments.
```
```suggestion
    override suspend fun getPicture(categoryName: String): CategorizedPicture? {
```